osu-ac/comparer.py

```diff
@@ -89,13 +94,21 @@ def compare(self, mode):
 
             for key in results:
                 self._print_result(results[key], key[0], key[1])
+            done += 1
+            rem = done % tenth
```

Comment:
make this and the other one a one liner, if(done % tenth == 0)

Reply:
oh you can't do that lol because you increment done

Reply:
you could use enumerate and not have to increment

Reply:
yeah wrap an enumerate call around the for loops and use that instead, also your stddevs loop logic is wrong - you never increment inside a loop. Wrap enumerate around the proper loop there; since we print results after all comparisons are done the progress bar will always finish printing there before any results print.